FCCee/FullSim/ALLEGRO/ALLEGRO_o1_v03/run_digi_reco.py

```diff
@@ -284,28 +286,30 @@
                                                     filterNoiseThreshold=0,
                                                     numRadialLayers=11,
                                                     scaleFactor=1 / 1000.,  # MeV to GeV
-                                                    OutputLevel=DEBUG)
-
-    # needs to be migrated!
-    #from Configurables import TubeLayerPhiEtaCaloTool
-    #barrelGeometry = TubeLayerPhiEtaCaloTool("EcalBarrelGeo",
-    #                                         readoutName=ecalBarrelReadoutNamePhiEta,
-    #                                         activeVolumeName="LAr_sensitive",
-    #                                         activeFieldName="layer",
-    #                                         activeVolumesNumber=12,
-    #                                         fieldNames=["system"],
-    #                                         fieldValues=[4])
+                                                    OutputLevel=INFO)
+
+    # barrel geometry tool is migrated to match ALLEGRO v3 segmentation
+    from Configurables import TubeLayerModuleThetaMergedCaloTool
+    barrelGeometry = TubeLayerModuleThetaMergedCaloTool("EcalBarrelGeo",
+                                             readoutName=ecalBarrelReadoutName,
+                                             activeVolumeName="LAr_sensitive",
+                                             activeFieldName="layer",
+                                             activeVolumesNumber=11,
```

Comment:
We commented in https://github.com/HEP-FCC/k4RecCalorimeter/pull/107 that the activeVolumesNumber property should be dropped